pkgs/by-name/ta/taktuk/package.nix

```diff
@@ -22,8 +22,8 @@ stdenv.mkDerivation {
     domain = "gitlab.inria.fr";
     owner = "taktuk";
     repo = "taktuk";
-    rev = "dcd763e389a414f540b43674cbc63752176f1ce3"; # does not tag releases
-    hash = "sha256-CerOBn1VDiKFLaW2WXv6wLxfcqy1H3dlF70lrequbog=";
+    rev = "e7126aeaa310d89c05dbe0629be422ee3eeb1c76"; # does not tag releases
```

Comment:
>does not tag releases

It appears upstream has provided a tag for version 3.7.8.
https://gitlab.inria.fr/taktuk/taktuk/-/tags/version-3.7.8

Reply:
Yes, we asked for it! I'm going to update the PR